pkgs/development/python-modules/textual-autocomplete/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, textual
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "textual-autocomplete";
+  version = "3.0.0-unstable-2024-07-10";
```

```suggestion
  version = "3.0.0a9";
```
https://pypi.org/project/textual-autocomplete/3.0.0a9/